src/js/api/config.js

```diff
@@ -75,7 +75,7 @@ define([
         __webpack_public_path__ = config.base;
         config.width  = _normalizeSize(config.width);
         config.height = _normalizeSize(config.height);
-        config.flashplayer = config.flashplayer || utils.getScriptPath('jwplayer.js') + 'jwplayer.flash.swf';
+        config.flashplayer = config.flashplayer || config.base + 'jwplayer.flash.swf';
```

Comment:
> Is your main use case self-hosted? 

Yes.

> Are you setting base in your config?

Yes, `base` is set correctly. `flashplayer` also needs to be set or it will not be found even though it is in the default location. I would like to avoid that.

> If bundling is the concern this should read:...

Isn't that redundant since `config.base` is already set to  `utils.getScriptPath('jwplayer.js')`?
https://github.com/jwplayer/jwplayer/blob/master/src/js/api/config.js#L72
https://github.com/jwplayer/jwplayer/blob/master/src/js/utils/playerutils.js#L42